cypress/integration/brewery/TableParameters-invalid_files.spec.js

```diff
@@ -8,6 +8,67 @@ import { DEFAULT_SCENARIOS_LIST } from '../../fixtures/stubbing/default';
 
 const CSV_INVALID_FILE_PATH = 'customers_invalid.csv';
 const XLSX_INVALID_FILE_PATH = 'customers_invalid.xlsx';
+const CSV_TOO_MUCH_PARAMETERS = 'customers_too_much_parameters.csv';
+const XLSX_TOO_MUCH_PARAMETERS = 'customers_too_much_parameters.xlsx';
+const CSV_ONLY_NAME_PARAMETER = 'customers_only_name_parameter.csv';
+const XLSX_ONLY_NAME_PARAMETER = 'customers_only_name_parameter.xlsx';
+
+const expectedErrorsFewCSV = [
+  { summary: 'Too few fields', loc: 'Line 1' },
+  { summary: 'Incorrect int value', loc: 'Line 2 , Column 2 ("age")' },
+  { summary: 'Incorrect bool value' },
+  { summary: 'Incorrect enum value' },
+  { summary: 'Incorrect date value' },
+  { summary: 'Incorrect number value' },
+  { summary: 'Incorrect int value' },
+  { summary: 'Incorrect bool value' },
+  { summary: 'Incorrect enum value' },
+  { summary: 'Incorrect date value' },
+  { summary: 'Incorrect number value' },
+];
+
+const expectedErrorsFewXLSX = [
+  { summary: 'Empty field', loc: 'Line 1, Column 3 ("canDrinkAlcohol")' },
+  { summary: 'Empty field', loc: 'Line 1, Column 4 ("favoriteDrink")' },
+  { summary: 'Incorrect int value', loc: 'Line 2 , Column 2 ("age")' },
+  { summary: 'Incorrect bool value' },
+  { summary: 'Incorrect enum value' },
+  { summary: 'Incorrect date value' },
+  { summary: 'Incorrect number value' },
+  { summary: 'Incorrect int value' },
+  { summary: 'Incorrect bool value' },
+  { summary: 'Incorrect enum value' },
+  { summary: 'Incorrect date value' },
+  { summary: 'Incorrect number value' },
+];
+
+const expectedErrorsMuchCSV = [
+  { summary: 'Empty field', loc: 'Line 1, Column 1 ("name")' },
+  { summary: 'Too many fields', loc: 'Line 1' },
+  { summary: 'Empty field', loc: 'Line 2, Column 1 ("name")' },
+  { summary: 'Too many fields', loc: 'Line 2' },
+  { summary: 'Too many fields', loc: 'Line 3' },
+  { summary: 'Too many fields', loc: 'Line 4' },
+  { summary: 'Empty field', loc: 'Line 6, Column 4 ("favoriteDrink")' },
+  { summary: 'Too few fields', loc: 'Line 6' },
+];
+
+const expectedErrorsMuchXLSX = [
+  { summary: 'Empty field', loc: 'Line 1, Column 1 ("name")' },
+  { summary: 'Too many fields', loc: 'Line 1' },
+  { summary: 'Empty field', loc: 'Line 2, Column 1 ("name")' },
+  { summary: 'Too many fields', loc: 'Line 2' },
+  { summary: 'Too many fields', loc: 'Line 3' },
+  { summary: 'Too many fields', loc: 'Line 4' },
+  { summary: 'Empty field', loc: 'Line 6, Column 4 ("favoriteDrink")' },
+];
```

Comment:
We can prevent writing twice the same data  by creating the "csv errors" array from a copy of the "xlsx errors", and adding the "Too few fields" item to this copy. Factorizing these arrays would help make the tests easier to maintain :slightly_smiling_face: